src/highdicom/sr/utils.py

```diff
@@ -177,13 +177,13 @@ def get_coded_value(item: Dataset) -> CodedConcept:
     )
 
 
-def get_coded_modality(item: Dataset) -> str:
+def get_coded_modality(sop_class_uid: str) -> Code:
```

Comment:
It looks like this is limited to datasets representing images. That should be mentioned in the docstring.